index.js

```diff
@@ -11,6 +11,7 @@ const minimist = require('minimist')
 const clc = require('cli-color')
 const flatCache = require('flat-cache')
 const urlsFromCrawling = require('./lib/getUrlsFromCrawler')
+const urlsFromFile = require('./lib/get-urls-from-file')
```

Comment:
I'm confused about the official recommendations on how we name the imported modules.
Should they be hyphen-deliminated or camelCase?
Either way, if we decide on one, i'll make everything else consistent.

Reply:
depends on your taste :-)

I usually prefer hyphens in filenames and camelCase for variables. But this is your project so I follow whatever style you prefer.

Reply:
I like the hyphen way, after this is merged, I will change the other ones. :)